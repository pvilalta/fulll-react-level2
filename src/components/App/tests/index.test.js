import React from 'react';
import { render } from '@testing-library/react';
import Component from '..';

describe('components | App | index', () => {
  it('should render component', () => {
    const { container } = render(<Component />);
    expect(container).toMatchSnapshot();
  });
});
