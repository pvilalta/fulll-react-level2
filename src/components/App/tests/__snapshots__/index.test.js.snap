// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components | App | index should render component 1`] = `
<div>
  <div
    class="App"
  >
    <div
      class="container"
    >
      <div
        class="SearchBar"
      >
        <form>
          <div
            class="searchBarBox"
          >
            <div
              class="searchBarTitle"
            >
              <label
                for="searchBar"
              >
                <span
                  class="searchBarTitleContent"
                >
                  Github profile finder
                </span>
              </label>
            </div>
            <div
              class="searchBarInput"
            >
              <input
                data-testid="searchBar"
                id="searchBar"
                name="searchBar"
                placeholder="Search..."
                type="text"
                value=""
              />
            </div>
            <button
              type="submit"
            />
          </div>
        </form>
        <div
          class="infoBox"
        />
      </div>
    </div>
  </div>
</div>
`;
